lib/Doctrine/Annotations/ImplicitlyIgnoredAnnotationNames.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations;
+
+/**
+ *  A list of annotations that are implicitly ignored during the parsing process.
+ *
+ *  All names are case sensitive.
+ */
+final class ImplicitlyIgnoredAnnotationNames
+{
+    private const Reserved = [
+        'Annotation' => true,
+        'Attribute'  => true,
+        'Attributes' => true,
+        /* Can we enable this? 'Enum' => true, */
```

Comment:
What about this question? This is from 5 years ago. I think it can be removed.

Reply:
Didn't really want to do side-effects during just _extracting_ the annotations. Also in 2.0 these reserved annotations will likely not be needed anymore.